README.md

```diff
@@ -26,49 +10,112 @@ This is a Java Spring Boot Web API reference application designed to "fork and c
+                                        toolchains file
+ -h,--help                              Display help information
+ -l,--log-file <arg>                    Log file where all build output
+                                        will go (disables output color)
+ -llr,--legacy-local-repository         Use Maven 2 Legacy Local
+                                        Repository behaviour, ie no use of
+                                        _remote.repositories. Can also be
+                                        activated by using
+                                        -Dmaven.legacyLocalRepo=true
+ -N,--non-recursive                     Do not recurse into sub-projects
+ -npr,--no-plugin-registry              Ineffective, only kept for
+                                        backward compatibility
+ -npu,--no-plugin-updates               Ineffective, only kept for
+                                        backward compatibility
+ -nsu,--no-snapshot-updates             Suppress SNAPSHOT updates
+ -ntp,--no-transfer-progress            Do not display transfer progress
+                                        when downloading or uploading
+ -o,--offline                           Work offline
+ -P,--activate-profiles <arg>           Comma-delimited list of profiles
+                                        to activate
+ -pl,--projects <arg>                   Comma-delimited list of specified
+                                        reactor projects to build instead
+                                        of all projects. A project can be
+                                        specified by [groupId]:artifactId
+                                        or by its relative path
+ -q,--quiet                             Quiet output - only show errors
+ -rf,--resume-from <arg>                Resume reactor from specified
+                                        project
+ -s,--settings <arg>                    Alternate path for the user
+                                        settings file
+ -t,--toolchains <arg>                  Alternate path for the user
+                                        toolchains file
+ -T,--threads <arg>                     Thread count, for instance 2.0C
+                                        where C is core multiplied
+ -U,--update-snapshots                  Forces a check for missing
+                                        releases and updated snapshots on
+                                        remote repositories
+ -up,--update-plugins                   Ineffective, only kept for
+                                        backward compatibility
+ -v,--version                           Display version information
+ -V,--show-version                      Display version information
+                                        WITHOUT stopping build
+ -X,--debug                             Produce execution debug output
+
+ ```
+
+## Run the Application
```

Comment:
This is the maven usage, but you need the ngsa-java usage.  Ping @kforeverisback (Kushal) and he can help you get the command to run it and print out the app usage.  I took a quick look.  There may be a bug, because I didn't see logic for handling "--help" or something like that to print usage.  If so, let's verify it and log it.

Reply:
Agree - seems to be a bug. `mvn spring-boot:run -Dspring-boot.run.arguments=--help` should print out the usage, and this is not the case. The code only prints the usage when passing in an invalid --log-level argument. @abassey2 can you plz create an item on triage?

For now, the right usage for docs should be

```    mvn clean spring-boot:run
 	-Dspring-boot.run.arguments=" --help
		--cpu.target.load
		--cpu.max.load
		--dry-run
		--log-level=<trace|info|warn|error|fatal>"
```

Reply:
I like how maven prints out its usage, maybe we can talk about doing something similar when we fix that bug